File: Beginner_Projects/Patterns/Alphabats/C++/P.cpp
#include <iostream>
#include <conio.h>
using namespace std;

int main()
{
    for (int row = 0; row < 7; row++)
    {
        for (int col = 0; col < 6; col++)
        {
            if ((row * col == 0 && col != 5) || (col == 5 && (row > 0 && row < 3) || (row == 3 && col < 4)))
            {
                cout << "*";
            }
       
            else
            {
                cout << " ";
            }
        }
        cout << endl;
    }
    getch();
}


//Created by Salfi Hacker Mansoor Bukhari

File: Beginner_Projects/Patterns/Alphabats/Python/P.py
for row in range(7):
    for col in range(6):
        if ((row * col == 0 and col != 5) or (col == 5 and (row > 0 and row < 3) or (row == 3 and col < 5))):
            print("*", end=" ")
        else:
            print(end="  ")
    print()
input()
